apps/condo/jest.config.js

```diff
@@ -1,4 +1,21 @@
+const conf = require('@core/config')
+
+/** @type {import('@jest/types').Config.InitialOptions} */
 module.exports = {
+    // this is not working in jest =( https://github.com/facebook/jest/issues/11500
+    // testTimeout: 1000000,
+    silent: true,   
+    // https://stackoverflow.com/questions/43864793/why-does-jest-runinband-speed-up-tests
+    maxWorkers: conf.IN_CI ? 1 : 4, 
+    reporters: [
+        'default',
+        [
+            'jest-stare',
```

Comment:
Could you please create a documentation file `docs/testing.md` and describe the usage of this plugin and the availability of `test-stare` directory for published applications in our servers.
PS: Do we really need this extra visuals?

Reply:
I've created custom reporter to hide this logs.
 https://github.com/MrFoxPro/jest-clean-reporter